src/common/containers/Play.js

```diff
@@ -67,13 +72,14 @@ export class Play extends Component {
     onJoinGame: func,
     onSpectateGame: func,
     onSendChatMessage: func,
-    onSelectDeck: func
+    onSelectDeck: func,
+    onSelectFormat: func
   };
 
   static baseUrl = '/play';
 
-  static urlForGameMode = (mode, deck = null) =>
-    deck ? `${Play.baseUrl}/${mode}/${deck.id}` : `${Play.baseUrl}/${mode}`;
+  static urlForGameMode = (mode, format = null, deck = null) =>
+    `${Play.baseUrl}/${mode}${format ? `/${format}` : ''}${deck ? `/${deck.id}` : ''}`;
```

Comment:
Oh man this nested interpolation kinda hurts me

Reply:
Let me try to make it nicer.